tsconfig.base.json

```diff
@@ -14,6 +14,7 @@
         "noUncheckedSideEffectImports": true,
         "types": ["node", "vitest", "vitest/globals"],
         "strictNullChecks": true,
+        "target": "ES2023", // Support top-level await
```

Comment:
Is this the latest target we can support on Node22? I think we might even get away with esnext but I'm not sure.